kubernetes/admission/test_fluxinterval.rego

```diff
@@ -0,0 +1,21 @@
+package library.kubernetes.validating.fluxinterval
+
+test_ten_minutes_allowed {
+	count(deny) == 0 with input as {"spec": {"template": {"spec": {"containers": [{"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=10m", "--sync-interval=10m"]}]}}}}
+}
+
+test_one_hour_allowed {
+	count(deny) == 0 with input as {"spec": {"template": {"spec": {"containers": [{"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=1h", "--sync-interval=1h"]}]}}}}
+}
+
+test_five_minutes_denied {
+	deny with input as {"spec": {"template": {"spec": {"containers": [{"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=5m", "--sync-interval=5m"]}]}}}}
+}
+
+test_five_minutes_denied_git_poll {
+	count(deny) == 1 with input as {"spec": {"template": {"spec": {"containers": [{"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=5m", "--sync-interval=10m"]}]}}}}
+}
+
+test_five_minutes_denied_sync_interval {
+	count(deny) == 1 with input as {"spec": {"template": {"spec": {"containers": [{"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=10m", "--sync-interval=5m"]}]}}}}
+}
```

Comment:
If you had a test with 2 images you can see the problem with iteration.  The containers below would be approved by the real-world policy, but the Rego above will reject it, I believe.
[
 {"image": "fluxcd/flux:1.20.2", "args": ["--git-poll-interval=10m", "--sync-interval=10m"]},
 {"image": "foo", "args": ["--git-poll-interval=1m"]}
]

Reply:
Thoughts on remove the `contains` check altogether?

Reply:
You still want the `contains` check that remains b/c it's ensuring it's a flux image.

To verify your new code is working as expected, I'd say it's worth adding a test with the 2 containers as shown in my comment.